src/content/blog/future-of-astro-zero-js-view-transitions.mdx

```diff
@@ -0,0 +1,106 @@
+    Sorry, your browser doesn't support embedded videos.
+  </video>
+  <figcaption class="text-center text-sm my-4">
+    <a href="https://x.com/astrodotbuild/status/1698534383970656497">Spotify clone</a> built with Astro View Transitions
+  </figcaption>
+</figure>
+ 
+If your browser does not support the latest cross-domain view transitions, then your website will still run and display content as expected. However, the animations will be disabled and page navigation will behave like any other navigation in your browser (without animation).
+
+## Upgrade
+
+To switch to the native new View Transition API, it’s as simple as adding the `@view-transition` at-rule to your website. Here’s all the CSS you need:
+
+```css
+@view-transition {
+  navigation: auto;
+}
+```
+
+To enable cross-document view transitions, both "ends" of the transition must include this rule. A shared page layout component can be a good place to add this to your codebase.
+
+If you were already using Astro View Transitions powered by our `<ViewTransitions />` component, then you are now safe to remove that component as well. This will remove the injected backwards-compatible JavaScript router from your page, reducing the overall complexity of your client-side code.
+
+
+```diff
+---
+- import { ViewTransitions } from "astro:transitions";
+---
+<head>
+-  <ViewTransitions fallback="none" />
++  <style>
++    @view-transition {
++      navigation: auto;
++    }
++  <style>
+</head>
+```
+
+If you would still like your view transitions to run in older browsers, you are also welcome to keep using our backwards-compatible `<ViewTransitions />` component. You can migrate whenever you are ready. We have no plans to remove support for the `<ViewTransitions />` component at this time.
+
+## What about transition:persist?
+
+[Persistent element transitions](https://docs.astro.build/en/guides/view-transitions/#maintaining-state) became a popular feature in Astro. The `transition:persist` directive told Astro to persist an element across navigation without interrupting or disrupting it's state. This enabled UI like audio and video players to continue to play across pages, like in <a href="https://twitter.com/charca">Maxi Ferreira's</a> <a href="https://astro-records.pages.dev/">music player demo</a>.
+
+Unfortunately, this feature is not yet supported by native cross-document navigation in the browser. Native support for persistent audio and video players may be coming  in the future, but in the meantime you can continue to use our `<ViewTransitions />` component if you need this additional functionality.
+
+### The Future of Astro
+
+We believe this is one of the most significant advancements to come out of Chromium and the browser spec process in recent memory. Following links across websites is a fundamental behavior of the web, and this represents the largest meaningful change in recent memory to how that behavior works.
```

Comment:
"Following links across websites" sounds like an odd turn of phrase, maybe? And if it's a fundamental behaviour, how is a brand new, meaningful change?

Reply:
haha good catch, I'll rewrite.